text/0034-telemetry-policies.md

```diff
@@ -0,0 +1,111 @@
+- Title: Telemetry Policies
+- Start Date: 2021-04-16
+- Specification PR: [#34](https://github.com/meilisearch/specifications/pull/34)
+- MeiliSearch Tracking-Issues: TBD
+
+# Telemetry Policies
+
+## 1. Feature Description and Interaction
+
+### I. Summary
+
+Telemetry concerns the possibility of depositing markers within a system in order to make segments of it. Quantitative data as data metrics are used to discover and verify user behavior.
+
+We would like to have more telemetry to better understand our users in order to improve their usage and feel with Meilisearch.
+
+### II. Motivation
+
+We need to make informed decisions based on data instead of instincts and gut feel. Lack of data does not say anything about how correct or wrong are our intuitions. Lack of data means we did not make testable hypothesis. We will now support ours decisions with data.
```

Comment:
```suggestion
We need to make informed decisions based on data instead of instincts and gut feel. Lack of data does not say anything about how correct or wrong are our intuitions. Lack of data means we did not make testable hypotheses. We will now support our decisions with data.
```